CHANGELOG.md

```diff
@@ -0,0 +1,16 @@
+# CHANGELOG
+Inspired from [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
+
+## [ UNRELEASED ]
+### Added
+- Github workflow for changelog verification
```

Comment:
Let's start with a format that includes a link to the PR?

Reply:
The only concern I have with that is it will become a 2 step process - 
1. Add changelog with dummy link and raise the PR
2. Update the changelog and commit again with the PR link

Let me check if there is an automated mechanism/property within the used GHA to achieve this.

Reply:
Working backwards from the end goal of users being able to find what the change was I think that's an acceptable overhead.